modules/tensor_mechanics/test/tests/cp_user_object/crysp_patch_recovery.i

```diff
@@ -0,0 +1,168 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 2
+  ny = 2
+  displacements = 'ux uy'
+[]
+
+[Variables]
+  [./ux]
+    block = 0
+  [../]
+  [./uy]
+    block = 0
+  [../]
+[]
+
+[AuxVariables]
+  [stress_xx_recovered]
+    order = FIRST
+    family = LAGRANGE
+  []
+  [stress_yy_recovered]
+    order = FIRST
+    family = LAGRANGE
+  []
+[]
+
+[Functions]
+  [./tdisp]
+    type = ParsedFunction
+    value = 0.01*t
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+    displacements = 'ux uy'
+    use_displaced_mesh = true
+  [../]
+[]
+
+[AuxKernels]
+  [stress_xx_recovered]
+    type = RankTwoAux
```

Comment:
This is not a big deal, but I would like to use a different name since  the AUX variable is taking  this name.

Reply:
sure, I can do that. We've normally used the same name for both the variable and the auxkernel in tensor mechanics, but I can change that convention for just this one test

Reply:
@fdkong - This is a convention and it's one we use in MOOSE too. I would really suggest that we do NOT break convention here. Thanks @sapitts for being willing to change, but also feel free to push back if it's something we do everywhere else. Sometimes certain members of the team aren't familiar with conventions downstream.

Reply:
Thanks for the clarification @permcody; I wasn't sure if there was a different between MOOSE framework vs modules conventions